.goreleaser.yml

```diff
@@ -0,0 +1,132 @@
+    ldflags:
+      - -X {{ .Env.REPO }}/internal/version.Version={{ if not .IsSnapshot }}v{{ end }}{{ .Version }}
+      - -X {{ .Env.REPO }}/internal/version.GitVersion={{ .Tag }}
+      - -X {{ .Env.REPO }}/internal/version.GitCommit={{ .FullCommit }}
+      - -X {{ .Env.REPO }}/internal/version.KubernetesVersion={{ .Env.K8S_VERSION }}
+    goarch:
+      - amd64
+      - arm64
+      - ppc64le
+      - s390x
+    goos:
+      - linux
+      - darwin
+    ignore:
+      - goos: darwin
+        goarch: arm64
+      - goos: darwin
+        goarch: ppc64le
+      - goos: darwin
+        goarch: s390x
+
+  # ansible-operator build steps
+  - id: ansible-operator
+    main: ./cmd/ansible-operator
+    binary: ansible-operator
+    env:
+      - CGO_ENABLED=0
+      - GO111MODULE=on
+      - REPO=github.com/operator-framework/operator-sdk
+    mod_timestamp: "{{ .CommitTimestamp }}"
+    asmflags:
+      - all=-trimpath={{ dir .ArtifactPath | dir | dir | dir }}
+    gcflags:
+      - all=-trimpath={{ dir .ArtifactPath | dir | dir | dir }}
+    ldflags:
+      - -X {{ .Env.REPO }}/internal/version.Version={{ if not .IsSnapshot }}v{{ end }}{{ .Version }}
+      - -X {{ .Env.REPO }}/internal/version.GitVersion={{ .Tag }}
+      - -X {{ .Env.REPO }}/internal/version.GitCommit={{ .FullCommit }}
+      - -X {{ .Env.REPO }}/internal/version.KubernetesVersion={{ .Env.K8S_VERSION }}
+    goarch:
+      - amd64
+      - arm64
+      - ppc64le
+      - s390x
+    goos:
+      - linux
+      - darwin
+    ignore:
+      - goos: darwin
+        goarch: arm64
+      - goos: darwin
+        goarch: ppc64le
+      - goos: darwin
+        goarch: s390x
+
+  # helm-operator build steps
+  - id: helm-operator
+    main: ./cmd/helm-operator
+    binary: helm-operator
+    env:
+      - CGO_ENABLED=0
+      - GO111MODULE=on
+      - REPO=github.com/operator-framework/operator-sdk
+    mod_timestamp: "{{ .CommitTimestamp }}"
+    asmflags:
+      - all=-trimpath={{ dir .ArtifactPath | dir | dir | dir }}
+    gcflags:
+      - all=-trimpath={{ dir .ArtifactPath | dir | dir | dir }}
+    ldflags:
+      - -X {{ .Env.REPO }}/internal/version.Version={{ if not .IsSnapshot }}v{{ end }}{{ .Version }}
+      - -X {{ .Env.REPO }}/internal/version.GitVersion={{ .Tag }}
+      - -X {{ .Env.REPO }}/internal/version.GitCommit={{ .FullCommit }}
+      - -X {{ .Env.REPO }}/internal/version.KubernetesVersion={{ .Env.K8S_VERSION }}
+    goarch:
+      - amd64
+      - arm64
+      - ppc64le
+      - s390x
+    goos:
+      - linux
+      - darwin
+    ignore:
+      - goos: darwin
+        goarch: arm64
+      - goos: darwin
+        goarch: ppc64le
+      - goos: darwin
+        goarch: s390x
+
+# Use most recent tag and short commit for snapshot version.
+snapshot:
+  name_template: "{{ .Tag }}"
+
+# We don't use archives, so skip creating them.
+archives:
+  - format: binary
+
+checksum:
+  name_template: 'checksums.txt'
```

Comment:
This would be net new functionality right? I have no problem whatsoever adding this, but I'd also be on board with disabling the checksum file generation.

Reply:
This is net-new, but simplifies signing the release since we'd only have to upload this file and its signature rather than a signature per binary.

Reply:
This file is just a checksum (I think sha256?). Not cryptographic signatures using a gpg key, which we still need.

Reply:
The `signs` block is commented out right now. Once a GPG key is created for CI and that section is uncommented, the `checksums.txt` file will be signed as `checksums.txt.asc` instead of each binary.